bogota-abril-2019/final_project/marvin-cely/tenis-match/src/components/FooterComponent/FooterPage.js

```diff
@@ -0,0 +1,69 @@
+import React from '../../../node_modules/react';
+import '../../css/FooterPage.css';
+import globantLogo from '../../images/globantLogo.png';
+
+class FooterPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      authorMessage: 'Creado por:',
```

Comment:
This should be a prop not a state since it's static content.

Reply:
Title component are props